javascript/javascript2/week2/movies/my-movies.js

```diff
@@ -0,0 +1,115 @@
+console.log("Number of selected movies : " + movies.length);
+
+// Movies with short titles
+
+const moviesWithShortTitle = movies
+    .filter((movie) => movie.title.length < 7)
+    .map((movie) => movie.title); // gives only the title
+console.log("\nShort titled movies()\n------------");
+console.log(moviesWithShortTitle);
+
+// movies with long titles
+const moviesWithLongTitle = movies
+    .filter((movie) => movie.title.length > 20)
+    .map((movie) => movie.title);
+console.log("\nLong titled movies()\n------------");
+console.log(moviesWithLongTitle);
+
+//Movies produced between 1980 and 1989
+
+let countMoviesProduced = 0;
+const moviesOf1980s = movies.filter(
+    (movie) => movie.year >= 1980 && movie.year <= 1989
+);
+countMoviesProduced++;
+console.log("\nNumber of Movies from 1980s()\n------------");
+console.log(moviesOf1980s.length);
+
+//console.log(moviesOf1980s); // array of movies from 1980s
+
+//Adding an extra tag
+
+movies[5].key = "tag"; //To add a new key called Tag
+movies[5].values = "newValue"; //To add a value pair based on rating
+
+const moviesWithRating = movies.map((movie) => {
+    if (movie.rating >= 7) return (movie.tag = "Good");
+    else if (movie.rating >= 4 && movie.rating <= 7)
```

Comment:
It should be less than 7
```suggestion
    else if (movie.rating >= 4 && movie.rating < 7)
```